libs/language-server/src/lib/ast/wrappers/value-type/index.ts

```diff
@@ -2,6 +2,8 @@
 //
 // SPDX-License-Identifier: AGPL-3.0-only
 
+export * from './util';
```

Comment:
I think there is a slight difference in those two exports that justify one being affected by the comment and one not.

In line 5, we just blindly export everything in the file (no matter if type or not) - it's not related to the WrapperFactory.
In line 14, we do care about type exports (so I added a comment that it is handled by the index.ts file) - it's related to the WrapperFactory.

It might be a code smell TBH.. I moved the value type utils one level up to the `wrappers/utils` directory ([9ae2670](https://github.com/jvalue/jayvee/pull/553/commits/9ae26707622f359b862f8440e13a37802a21399a)). Hope that makes it clearer :)